src/styles/styles.css

```diff
@@ -17,6 +17,33 @@
 .navbar__item:hover::before {
     @apply opacity-100;
 }
+
+/* Loader */
+.lds-dual-ring {
+    display: inline-block;
+    width: 24px;
+    height: 24px;
+}
+.lds-dual-ring:after {
+    content: ' ';
+    display: block;
+    width: 24px;
+    height: 24px;
+    margin: 8px;
+    border-radius: 50%;
+    border: 2px solid #fff;
+    border-color: #fff transparent #fff transparent;
+    animation: lds-dual-ring 1.2s linear infinite;
+}
+@keyframes lds-dual-ring {
+    0% {
+        transform: rotate(0deg);
+    }
+    100% {
+        transform: rotate(360deg);
+    }
+}
+
 /* width */
```

Comment:
Should we follow the same way of applying properties as above/rest of this CSS file (using tailwind and @apply )?

Reply:
Actually, I just copy/paste that chunk of code. Maybe you'll want another loading button or just a loader somewhere. Those classes could be completely removed if we do it another way. That's why I let it this way. ;)